src/components/SeedFundReports/ReportFormAnnual.tsx

```diff
@@ -85,7 +84,9 @@ export default function ReportFormAnnual(props: ReportFormAnnualProps): JSX.Elem
   );
   const [socialImpact, setSocialImpact] = useState(report.annualDetails?.socialImpact ?? '');
 
-  const [sdgList, setSdgList] = useState(report.annualDetails?.sustainableDevelopmentGoals.map((g) => g.goal) ?? []);
+  const [sdgList, setSdgList] = useState(
+    report.annualDetails?.sustainableDevelopmentGoals.map((g) => g.goal.toString()) ?? []
+  );
```

Comment:
The goals from the BE can now come back as integers instead of strings, but the schema definition for goal is still string. This `toString()` addition ensures indices are found when searching the list for a specific SDG's string.